RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -157,7 +167,7 @@ private extension CustomerCenterView {
         let accentColor = Color.from(colorInformation: configuration.appearance.accentColor,
                                      for: self.colorScheme)
 
-        CompatibilityNavigationStack {
+        CompatibilityNavigationStack(isInNavigationStack: isInNavigationStack) {
             destinationContent(configuration: configuration)
         }
```

Comment:
```suggestion
        if isInNavigationStack { 
             CompatibilityNavigationStack(isInNavigationStack: isInNavigationStack) {
                 destinationContent(configuration: configuration)
             }
         } else {
             destinationContent(configuration: configuration)
         }
```

Reply:
Btw, since the suggested change already uses `isInNavigationStack` in the `if`, then `CompatibilityNavigationStack` init would not receive this `bool` anymore